web/stories/Page.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```

Comment:
actually this looks generated also, should just remove the generated stuff and stick to what youre using (`Button` in this case)